app/src/main/java/org/p2p/wallet/feerelayer/interactor/FeeRelayerAccountInteractor.kt

```diff
@@ -33,13 +34,15 @@ class FeeRelayerAccountInteractor(
     suspend fun getRelayInfo(): RelayInfo = withContext(Dispatchers.IO) {
```

Comment:
why you don't use serviceScope instead of Dispatchers.IO

Reply:
Why I should use serviceScope here?